system-variables.md

```diff
@@ -1566,11 +1629,29 @@ explain select * from t where age=5;
 
 ### tidb_pprof_sql_cpu <span class="version-mark">New in v4.0</span>
 
-- Scope: INSTANCE
+- Scope: GLOBAL
+- Persists to cluster: No
 - Default value: `0`
 - Range: `[0, 1]`
 - This variable is used to control whether to mark the corresponding SQL statement in the profile output to identify and troubleshoot performance issues.
 
+### tidb_prepared_plan_cache_memory_guard_ratio <span class="version-mark">New in v6.1.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Default value: `0.1`
+- This setting is used to prevent the tidb.toml option `performance.max-memory` from being exceeded. When `max-memory` * (1 - `tidb_prepared_plan_cache_memory_guard_ratio`) is exceeded, the elements in the LRU are removed.
+- This setting was previously a tidb.toml option (`prepared-plan-cache.memory-guard-ratio`), but changed to a system variable starting from TiDB 6.1.
+
+### tidb_prepared_plan_cache_size <span class="version-mark">New in v6.1.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Default value: `100`
+- Range: `[1, 100000]`
```

Comment:
Align with https://github.com/pingcap/docs-cn/pull/9553
```suggestion
- Range: `[0, 2147483647]`
```

Reply:
The minimum value is 1, see the source code: https://github.com/pingcap/tidb/blob/dce5064e9e15d0b17c36f73e21fc71bcaa419e90/sessionctx/variable/sysvar.go#L767

Reply:
Thanks. I'll correct the Chinese doc.